Example/Podfile

```diff
@@ -1,8 +1,8 @@
 # Uncomment the next line to define a global platform for your project
-# platform :ios, '9.0'
+ platform :ios, '10.0'
 
 target 'TestCountryPickerFramework' do
   # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
   use_frameworks!
-  pod 'SKCountryPicker' 
+  pod 'SKCountryPicker', :path => '~/Desktop/Projects/CountryPicker/CountryPicker/'
```

Comment:
@SURYAKANTSHARMA You may have to revert this line after testing. It was modified with the intention of modifying the library methods.

Reply:
ok @Sharkesm